src/samplers/tcp/stats.rs

```diff
@@ -30,6 +35,114 @@ counter_with_histogram!(
     "tcp/transmit/retransmit",
     "number of TCP segments retransmitted"
 );
+counter_with_histogram!(
+    TCP_CONN_ACTIVE_OPEN,
+    TCP_CONN_ACTIVE_OPEN_HISTOGRAM,
+    "tcp/connection/active_open",
+    "number of TCP active opens"
+);
+counter_with_histogram!(
```

Comment:
Where are these two histograms being used?

Reply:
Whoops. Wasn't supposed to include those in this PR.